.circleci/config.yml

```diff
@@ -911,6 +911,9 @@ jobs:
       folder:
         type: string
         default: ""
+      currents-record:
+        type: string
+        default: ""
```

Comment:
A personal preference and a nit:
This could be a boolean and then later in the command itself you could pass `"--key $CURRENTS_KEY"` instead of storing it as a string in the parameter.

There is technically no big difference, but it might make code more readable.